src/main/java/cafeboard/Board/BoardController.java

```diff
@@ -0,0 +1,32 @@
+package cafeboard.Board;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class BoardController {
+    private final BoardService SERVICE;
```

Comment:
상수는 대문자로 표기하긴 하는데, 얘는 `final`이 붙었지만 상수가 아니라 소문자로 적어요.
```suggestion
    private final BoardService boardService;
```